mps_youtube/commands/config.py

```diff
@@ -16,14 +16,14 @@ def showconfig():
         has_temps = has_temps or val.temp_value is not None
 
     width -= 27
-    s = f"  %s%-{longest_key}s%s : %-{longest_val+1}s"
+    s = "  %s%-{0}s%s : %-{1}s".format(longest_key, longest_val+1)
 
     if has_temps:
         width -= longest_val + 5
-        out = (f"  %s%-{longest_key}s   %-{longest_val}s %s%s%s\n") % (
+        out = ("  %s%-" + str(longest_key) + "s   %-" + str(longest_val) + "s %s%s%s\n") % (
```

Comment:
It seems awkward to mix string concatenation and `%` here. It would be better just use %, or maybe switch to `.format()`.

This was an issue before this PR, but with concatenation makes it even worse.

Reply:
I was bit lazy while writing this, sorry for that. I guess it's better now.

Tested on python 3.4.3 and it works!